README.md

```diff
@@ -116,6 +116,13 @@ Follow the given instructions for Login into the app.
    Fill up the things it asked to and then Login into the app.
 
 ## Environment Variables
+Use .env.template file to make a new .env file. You can read about environment variables used in this [here](https://github.com/anitab-org/anitab-forms-backend/blob/develop/main/settings.py).
+
+```
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<your-email-address>
+EMAIL_PASSWORD=<your-api-key>
+```
```

Comment:
```suggestion
Make sure you create `.env` using `.env.template` and update the values of corresponding environment variables or make sure you exported the following [environment variables](#Environment-Variables).
```
Also, add the above-mentioned change as the 6th point i.e. after `pip install -r requirements.txt` instead of this section.

Reply:
@codesankalp, Do I need to include the three environment variables in the 6th point or not?